File: Level 1/로또의 최고 순위와 최저 순위/solution.py
def solution(lottos, win_nums):
    rank = [6, 6, 5, 4, 3, 2, 1]
    min_count = len([num for num in lottos if num in win_nums])
    zero_count = lottos.count(0)

    max_count = min_count + zero_count

    return [rank[max_count], rank[min_count]]

File: Level 1/로또의 최고 순위와 최저 순위/solution.js
function solution(lottos, win_nums) {
  const rank = [6, 6, 5, 4, 3, 2, 1];
  const zeroCount = lottos.filter((num) => num === 0).length;
  const minCount = lottos.filter((num) => win_nums.includes(num)).length;

  const maxCount = minCount + zeroCount;

  return [rank[maxCount], rank[minCount]];
}
